public/scripts/client.js

```
@@ -1,28 +1,25 @@
-  const renderTweets = function(tweets) {
+$(document).ready(function () {
+  const renderTweets = function (tweets) {
     for (tweet of tweets) {
       const $tweet = createTweetElement(tweet);
       $("#tweet-container").prepend($tweet);
     }
   };
 
   // Prevent XSS with Esaping
-  const escape = function(str) {
+  const escape = function (str) {
```

Biome (1.9.4)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)